nerdlandbot/commands/reminder.py

```diff
@@ -0,0 +1,98 @@
+import typing
+import asyncio
+import discord
+from discord.ext import commands
+import datetime
+
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.constants import *
+
+
+class Reminder(commands.Cog, name="Simple Reminder"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="reminder", aliases=["remind_me"], brief="reminder_brief", usage="reminder_usage", help="reminder_help")
+    async def reminder(self, ctx: commands.Context, time: typing.Optional[str] = None, *, message: typing.Optional[str] = None):
+        # If the user doesn't enter a time-format, let him know
+        try:
+            time_int = int(time)
+        except:
+            msg = translate("reminder_err_no_int", await culture(ctx))
+            title = translate("reminder_err_title", await culture(ctx))
+            embed = discord.Embed(
+                description=msg,
+                color=NOTIFY_EMBED_COLOR
+            )
+            return await ctx.send(embed=embed)
+
+        # If the time is longer than a day
+        if time_int >= MAX_REMINDER_TIME:
+            msg = translate("reminder_err_too_long", await culture(ctx))
+            title = translate("reminder_err_title", await culture(ctx))
+            embed = discord.Embed(
+                title=title,
+                description=msg,
+                color=NOTIFY_EMBED_COLOR
+            )
+            return await ctx.send(embed=embed)
+
+        # If the time is shorter than 5 minutes
+        if time_int <= MIN_REMINDER_TIME:
+            msg = translate("reminder_err_too_short", await culture(ctx))
+            title = translate("reminder_err_title", await culture(ctx))
+            embed = discord.Embed(
+                title=title,
+                description=msg,
+                color=NOTIFY_EMBED_COLOR
+            )
+            return await ctx.send(embed=embed)
+        
+        # Logic to determine the time the user wants to be reminded
+        current_time = datetime.datetime.now()
+        time_seconds = time_int * REMINDER_CONVERT_TO_MINUTES
+        remind_time = current_time + datetime.timedelta(0, time_seconds)
```

Comment:
I would change this to `remind_time = current_time + datetime.timedelta(minutes=time_int)` to make it clearer what it actually does. This way it is also not needed to convert it to seconds